resources/configurations/descriptions/participants/local_discovery_server.yaml

```diff
@@ -1,9 +1,9 @@
 
-####################################
-# Simple Participant Configuration #
-####################################
+####################################################
+# Local Discovery Server Participant Configuration #
+####################################################
 
-# Simple participant for local communication
+# Local Discovery Server participant
```

Comment:
Update  the `DiscoveryServerGuid` and ListeningAddresses` tags in this file.

Reply:
It has been decided to remove the whole ``descriptions`` directory as its purpose is no longer required.